spec/abstract-jobs.html

```diff
@@ -162,18 +171,19 @@ <h1>Liveness</h1>
     <emu-clause id="sec-weakref-execution">
       <h1>Execution</h1>
 
-      <p>At any time, if an object _obj_ is not live, an ECMAScript implementation may perform the following steps atomically:</p>
+      <p>At any time, if a set of objects _S_ is not live, an ECMAScript implementation may perform the following steps atomically:</p>
```

Comment:
is the intention that none, or all, of the following steps are executed? Meaning, the overall set of steps is optional, but no single step is?

Reply:
That is correct.